app/src/main/java/com/sebastianvm/mango/ui/example/ExampleViewModel.kt

```diff
@@ -23,3 +31,15 @@ class ExampleViewModel(
         }
     }
 }
+
+
+@InstallIn(ViewModelComponent::class)
+@Module
+object InitialExampleStateModule {
+
+    @Provides
+    @ViewModelScoped
+    fun initialExampleStateProvider(): ExampleState {
+        return ExampleState()
+    }
+}
```

Comment:
do we have to do this whole block for every dependency? or all part of this object here

Reply:
We need to do this for most dependencies. This is essentially how we're telling hilt how to instantiate the dependencies